File: C#/455_AssignCookies.cs
public class Solution {
    public int FindContentChildren(int[] g, int[] s) {
        Array.Sort(g);
        Array.Sort(s);
        int content = 0;
        int gi = 0, si = 0;
        while (gi < g.Length && si < s.Length) {
            if (g[gi] <= s[si]){
                content++;  
                gi++;
            } 
            si++;
            
        }
        return content;
    }
}

File: Python/455_AssignCookies.py
class Solution:
    def findContentChildren(self, g: List[int], s: List[int]) -> int:
        g.sort()
        s.)
        k, c, happyKids = 0,0,0
        while k < len(g) and c < len(s):
            if g[k] <= s[c]:
                k = k + 1
                happyKids = happyKids + 1
            c = c + 1
        return happyKids
        
